### ISSUE TITLE
Xamarin.iOS build error 

### ISSUE THREAD

**Issue by charlesroddie** [verybadcat/CSharpMath#19]
iOS build error reported in automated build after updating from pre7 to pre9. Still happening in pre10.
```
MTOUCH : error MT2101: Can't resolve the reference 'System.Boolean CSharpMath.WarningException::get_DisableWarnings()', referenced from the method 'System.Boolean CSharpMath.Settings::get_DisableWarnings()' in 'CSharpMath, Version=0.1.0.0, Culture=neutral, PublicKeyToken=null'. [/Users/vsts/agent/2.139.1/work/1/s/Platforms/iOS/50iOS.csproj]
```
@Happypig375  @j-sauer

**Comment by Happypig375** [comment]
The cause for this is I forgot to push the pre9 package for CSharpMath to NuGet. Please explicitly install CSharpMath 0.1.0-pre10 for now. The next release, 0.1.0-rc.0 (won't be pre.11 as pre < pre1, won't be rc.1 as there are still features to add), should be ranked higher than pre9 so this should be fixed when it is out.

**Comment by Happypig375** [comment]
Has this been resolved as of 0.1.0-rc.0?

**Comment by charlesroddie** [comment]
@FoggyFinder will test in a few hrs

**Comment by FoggyFinder** [comment]
> Has this been resolved as of 0.1.0-rc.0?

Yep, it works 🎉

**Comment by Happypig375** [comment]
:hooray:
